physics/GFS_DCNV_generic.meta

```diff
@@ -25,6 +25,14 @@
   type = logical
   intent = in
   optional = F
+[qdiag3d]
+  standard_name = flag_tracer_diagnostics_3D
```

Comment:
Isn't it just specific humidity tendencies instead of all tracers?

Reply:
No.  It includes ozone tendencies.